src/modules/returns/lib/return-helpers.js

```diff
@@ -310,7 +294,25 @@ const applyMeterReadings = (data, formValues) => {
   }, input);
 
   updated.lines = readings.lines;
-  return set(updated, 'meters[0].readings', omit(formValues, 'csrf_token'));
+  set(updated, 'meters[0].startReading', formValues.startReading);
+  return set(updated, 'meters[0].readings', omit(formValues, ['csrf_token', 'startReading']));
+};
+
+const applyMeterReset = (data, formValues) => {
+  const { meterReset } = formValues;
+  const updated = cloneDeep(data);
+
+  if (meterReset) {
+    set(updated, 'reading.method', 'abstractionVolumes');
+    const meters = updated.meters || [];
+    for (let meter of meters) {
+      delete meter.readings;
```

Comment:
Just checking - do we need to delete these properties or just set them back to null, empty arrays etc.